src/Symfony/Component/HttpClient/Tests/TraceableHttpClientTest.php

```diff
@@ -80,4 +82,21 @@ public function testItResetsTraces()
         $sut->reset();
         $this->assertCount(0, $sut->getTracedRequests());
     }
+
+    public function testStream()
+    {
+        $response = new MockResponse('Hi !');
+
+        $client = $this->createMock(HttpClientInterface::class);
+        $client
+            ->expects($this->once())
```

Comment:
would it be possible to replace these mocks by a real HTTP calls instead? We have a running HTTP server for this purpose.
See https://twitter.com/frankdejonge/status/1242123354292846607 about mocks :)

Reply:
Great ! I didn't see the running HTTP server 🙂Updated.